rocky/rocky/views/mixins.py

```diff
@@ -380,21 +379,17 @@ def get_children_input_oois(children_reports: list[Report]) -> set[str]:
         return {input_ooi for child_report in children_reports for input_ooi in child_report.input_oois}
 
     @staticmethod
-    def report_type_summary(reports: list[Report]) -> dict[str, int]:
+    def report_type_summary(reports: list[Report]) -> dict[str, int] | None:
         """
         Calculates per report type how many objects it consumed.
         """
 
         summary: dict[str, int] = {}
         report_types: set[str] = {report.report_type for report in reports}
 
-        for report_type in report_types:
-            summary[report_type] = len(
-                {
-                    itertools.chain.from_iterable(
-                        [report.input_oois for report in reports if report_type == report.report_type]
-                    )
-                }
+        for report_type in sorted(report_types):
```

Comment:
Why are the report_types sorted?

Reply:
For readability it helps when viewing the summary